From 44afd5885faa1d13d0c7f3a53b3529b8b9349f18 Mon Sep 17 00:00:00 2001
From: rnaidu <ramya.v.naidu@intel.com>
Date: Wed, 29 Apr 2020 21:02:24 +0530
Subject: [PATCH] Fixing the compilation issue after AOSP rebase

ClangDefaultVersion is using wrong path.
Hence changed the default version to clang-r353983c

Tracked-On: OAM-90872
Signed-off-by: rnaidu <ramya.v.naidu@intel.com>
---
 cc/config/global.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cc/config/global.go b/cc/config/global.go
index 7c7b47a5..815c31d8 100644
--- a/cc/config/global.go
+++ b/cc/config/global.go
@@ -122,7 +122,7 @@ var (
 
 	// prebuilts/clang default settings.
 	ClangDefaultBase         = "prebuilts/clang/host"
-	ClangDefaultVersion      = "clang-r353983c1"
+	ClangDefaultVersion      = "clang-r353983c"
 	ClangDefaultShortVersion = "9.0.3"
 
 	// Directories with warnings from Android.bp files.
-- 
2.17.1

